components/ClaimDetails.tsx

```diff
@@ -15,7 +15,17 @@ export const ClaimDetails: React.FC<ClaimDetailsProps> = ({
   lastPaymentIssued,
   extensionType,
 }) => {
-  const { t } = useTranslation('common')
+  const { t } = useTranslation(['common', 'claim-status'])
+
+  let extension: JSX.Element | null = null
+  if (extensionType) {
+    extension = (
+      <div className="col-6">
+        <div />
+        <InfoField loading={loading} label={t('claim-details.extension-type')} text={t(extensionType)} />
```

Comment:
Wait how do these work?! You imported `claim-status` (presumably a bug), but the translations are still working!

Reply:
Whoa! That was a bug. I meant to import `claim-details` but maybe it doesn't need to explicitly import here because it's imported in `index.tsx`...?

Reply:
ooo maybe! we do need to get access to `t` here, but maybe there's some trick

Reply:
From https://github.com/isaachinman/next-i18next#4-declaring-namespace-dependencies:

> Note: useTranslation provides namespaces to the component that you use it in. However, serverSideTranslations provides the total available namespaces to the entire React tree and belongs on the page level. Both are required.

In my testing it seems like I can do `const { t } = useTranslation()` in `ClaimDetails.tsx` and the translations actually work fine, but because the documentation says it's required to pass in the namespace to `useTranslation()`, it seems safer to just do that. I'll fix this bug.

Reply:
I take it back. I'm not sure what was going on with my testing yesterday, but we do need `const { t } = useTranslation(['common', 'claim-details'])`in `ClaimDetails.tsx` for the translation strings to work.

Reply:
I've noticed some issues around hot reloading and translation files - might be that!